modules/renter/contractor/contractor.go

```diff
@@ -135,10 +130,7 @@ func (c *Contractor) Contracts() []modules.RenterContract {
 
 // ContractUtility returns the utility fields for the given contract.
 func (c *Contractor) ContractUtility(id types.FileContractID) (modules.ContractUtility, bool) {
-	c.mu.RLock()
-	utility, exists := c.contractUtilities[c.resolveID(id)]
-	c.mu.RUnlock()
-	return utility, exists
+	return c.contractUtility(id)
```

Comment:
convention violation - need to call `contractUtility` holding a lock. If there's no locking, it needs to be called `managedContractUtility`

Reply:
Can we make it `staticContractUtility` ? It doesn't really matter if you hold the contractor lock or not.